README.md

```diff
@@ -137,7 +137,7 @@ Klaviyo.setEmail("robin@example.com")
 ### Tracking Events
 
 The SDK also provides tools for tracking analytics events to the Klaviyo API.
-A list of common Klaviyo-defined event names is provided in [EventType](https://klaviyo.github.io/klaviyo-android-sdk/sdk/analytics/com.klaviyo.analytics.model/-event-type/), or
```

Comment:
I added versioning to the dokka docs, but I haven't figured out how to make github pages point to more than 1 level deep sub-directory, so for the moment I just need to put the version string in the URL explicitly.